package.json

```diff
@@ -7,7 +7,7 @@
     "@commitlint/config-conventional": "^7.5.0",
     "commitizen": "^3.0.4",
     "husky": "^1.3.1",
-    "lerna": "^3.11.0",
+    "lerna": "^3.4.3",
```

Comment:
is that an artefact or deliberate decision?

Reply:
good catch! was an artefact..